waiter/project.clj

```diff
@@ -111,11 +111,12 @@
                  [slingshot "0.12.2"]
                  [try-let "1.3.1"
                   :exclusions [org.clojure/clojure]]]
-  :eftest {:report clojure.test/report}
+  :eftest {:report clojure.test/report
+           :thread-count (or (some-> (System/getenv "LEIN_TEST_THREADS") Integer/valueOf)
```

Comment:
Should we do `parseInt` instead of `valueOf`?
`availableProcessors` also returns an `int`.